apps/worker/src/app/workflow/usecases/execute-bridge-job/execute-bridge-job.usecase.ts

```diff
@@ -193,99 +193,48 @@ export class ExecuteBridgeJob {
   }: Omit<ExecuteBridgeRequestCommand, 'afterResponse' | 'action' | 'retriesLimit'> & {
     job: JobEntity;
   }): Promise<ExecuteOutput> {
-    try {
-      return this.executeBridgeRequest.execute({
-        statelessBridgeUrl,
-        event,
-        action: PostActionEnum.EXECUTE,
-        searchParams,
-        afterResponse: async (response) => {
-          const body = response?.body as string;
-
-          if (response.statusCode >= 400) {
-            let rawMessage: Record<string, unknown>;
-            try {
-              rawMessage = JSON.parse(body);
-            } catch {
-              Logger.error(`Unexpected body received from Bridge: ${body}`, LOG_CONTEXT);
-              rawMessage = {
-                error: `Unexpected body received from Bridge: ${body}`,
-              };
-            }
-            const createExecutionDetailsCommand: CreateExecutionDetailsCommand = {
-              ...CreateExecutionDetailsCommand.getDetailsFromJob(job),
-              detail: DetailEnum.FAILED_BRIDGE_RETRY,
-              source: ExecutionDetailsSourceEnum.INTERNAL,
-              status: ExecutionDetailsStatusEnum.WARNING,
-              isTest: false,
-              isRetry: false,
-              raw: JSON.stringify({
-                url: statelessBridgeUrl,
-                statusCode: response.statusCode,
-                retryCount: response.retryCount,
-                message: response.statusMessage,
-                ...(body && body?.length > 0 ? { raw: rawMessage } : {}),
-              }),
+    return this.executeBridgeRequest.execute({
+      statelessBridgeUrl,
+      event,
+      action: PostActionEnum.EXECUTE,
+      searchParams,
+      afterResponse: async (response) => {
+        const body = response?.body as string;
+
+        if (response.statusCode >= 400) {
+          let rawMessage: Record<string, unknown>;
+          try {
+            rawMessage = JSON.parse(body);
+          } catch {
+            Logger.error(`Unexpected body received from Bridge: ${body}`, LOG_CONTEXT);
+            rawMessage = {
+              error: `Unexpected body received from Bridge: ${body}`,
             };
-
-            await this.createExecutionDetails.execute(createExecutionDetailsCommand);
           }
-
-          return response;
-        },
-        workflowOrigin,
-        environmentId,
-      }) as Promise<ExecuteOutput>;
-    } catch (error: any) {
-      Logger.error(error, 'Error sending Bridge request:', LOG_CONTEXT);
-
-      let raw: { retryCount?: number; statusCode?: number; message: string; url?: string };
-
-      if (error.response) {
```

Comment:
This code path was never executed with recent changes to the Bridge error handling, where the `got` error is now captured and a NestJS exception is thrown. It's now redundant (similar to the removal of the extra health check log)